content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -77,58 +78,62 @@ Deletion removes both the `VolumeSnapshotContent` object from the Kubernetes API
 
 ## Volume Snapshot Contents
 
-Each VolumeSnapshotContent contains a spec, which is the specification of the volume snapshot.
+Each VolumeSnapshotContent contains a spec and status, which is the specification and status of the volume snapshot content. In dynamic provisioning, the `VolumeSnapshotContent` object is created by the snapshot common controller.
 
 ```yaml
-apiVersion: snapshot.storage.k8s.io/v1alpha1
+apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshotContent
 metadata:
-  name: new-snapshot-content-test
+  name: snapcontent-72d9a349-aacd-42d2-a240-d775650d2455
 spec:
-  snapshotClassName: csi-hostpath-snapclass
+  deletionPolicy: Delete
+  driver: hostpath.csi.k8s.io
   source:
-    name: pvc-test
-    kind: PersistentVolumeClaim
-  volumeSnapshotSource:
-    csiVolumeSnapshotSource:
-      creationTime:    1535478900692119403
-      driver:          csi-hostpath
-      restoreSize:     10Gi
-      snapshotHandle:  7bdd0de3-aaeb-11e8-9aae-0242ac110002
+    volumeHandle: ee0cfb94-f8d4-11e9-b2d8-0242ac110002
+  volumeSnapshotClassName: csi-hostpath-snapclass
+  volumeSnapshotRef:
+    name: new-snapshot-test
+    namespace: default
+    uid: 72d9a349-aacd-42d2-a240-d775650d2455
 ```
 
-### Class
+For pre-provisioned snapshots, the admin is responsible for creating the `VolumeSnapshotContent` object.
```

Comment:
```suggestion
For pre-provisioned snapshots, you (as cluster administrator) are responsible for creating the `VolumeSnapshotContent` object.
```
per [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you).

Reply:
Done